### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jamietotten**
Recent updates for MINDBODY pre-christmas push

### CODE REVIEW

File: docs/welcome.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Lightrail enables you to build Customer Wallet solutions where all of the the promotions, gift cards, and loyalty points are associated with that customer’s identity. Lightrail believes that With this you can create frictionless payment experiences. 
+
+#### Checkout
+The Checkout API Endpoint allows you to submit a customer's shopping cart for payment processing. Lightrail will calculate the subtotal from the line items in the shopping cart, apply any Lightrail discounts or pre-paid sources, and finally charge the credit card all in a single request. Lightrail handles the complexity of charging these sources in the correct order creating a seamless payment experience for your customer.
+
+
+#### Getting Started
+See our examples below to get started with:
+
+- [Promotions](#use-cases/promotions)
```
Comment (Tim-Jordison) [inline docs/welcome.md]: Should this list be the same as the nav?
- accounts
- promotions
- drop-in gift cards

Conceivably should promotions come first?

File: docs/currency-and-points.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Currency and Points
+
+Lightrail supports any currency including custom currencies you create.
```
Comment (jamietotten) [inline docs/currency-and-points.md]: "any currency" seems so broad and vague. Should we say "any ISO currency..." and link to ISO def or something?

File: docs/object-model.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Object Model
+
+<p class="intro">Lightrail’s API and web app enables you to create and mange all types of customer value: promotions, gift cards, referral credits, refund credits, and loyalty points. Within Lightrail, these are stored as what we call a Value.</p>
+
+### Values
+
+Values are the core financial object in Lightrail. Values can have a `balance` or a `balanceRule`, which can be used to represent percentage discounts or more sophisticated promotions. See our Balance Rule Documentation for more information.
```
Comment (jamietotten) [inline docs/object-model.md]: "See our Balance Rule Documentation" -> link to `#use-cases/redemption-rules-and-balance-rules`

File: docs/object-model.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+<p class="intro">Lightrail’s API and web app enables you to create and mange all types of customer value: promotions, gift cards, referral credits, refund credits, and loyalty points. Within Lightrail, these are stored as what we call a Value.</p>
+
+### Values
+
+Values are the core financial object in Lightrail. Values can have a `balance` or a `balanceRule`, which can be used to represent percentage discounts or more sophisticated promotions. See our Balance Rule Documentation for more information. 
+
+Values have a `usesRemaining` property indicating how many times they can be used. Many promotions can only be used once, whereas a gift card, can be used indefinitely until its balance is used up. 
+
+Values can also have `redemptionRules` which enable sophisticated promotions. For example, you may want a promotion to apply to a particular product SKU or the promotion to apply only if the subtotal is above a certain amount. See our Redemption Rule Documentation for more information.
```
Comment (jamietotten) [inline docs/object-model.md]: "See our Redemption Rule Documentation" -> link to `#use-cases/redemption-rules-and-balance-rules`

File: docs/wallet.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Wallet
+
+<p class="intro">Lightrail enables the concept of a Wallet within your application or service. A Wallet is a framework for thinking about the many types of customer value attached to one of your users that will be available for them to spend when making a payment.</p>
+
+<table class="no-border">
+    <tr>
+        <td>
+            <img alt="wallet" style="width: 200px" src="https://raw.githubusercontent.com/Giftbit/Lightrail-API-V2-Docs/development/docs/assets/wallet.png" />
```
Comment (jamietotten) [inline docs/wallet.md]: Update image path to `master`

File: docs/welcome.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+Lightrail’s API enables two primary capabilities that enable customer value as a secure, flexible, and effective tool for your business and growth team. Those two capabilities are:
+
+<ol>
+<li>a digital wallet for each of your customers, and</li> 
+<li>a checkout integration that simplifies multi-tender payments with rule-based customer value.</li>
+</ol> 
+
+#### Who Is Lightrail For? 
+
+Lightrail is an API-first tool that allows you to build-in and customize your payment experience. Lightrail is flexible and supports single brand e-commerce requirements and also supports more sophisticated marketplace requirements.
```
Comment (jamietotten) [inline docs/welcome.md]: I'd suggest changing "allows you to build-in and customize your payment experience" to "requires you to build-in and allows you to customize your payment experience". But I'm easy here, it just seems more accurate.

File: docs/object-model.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+### Programs
+
+#### Creating and Organizing Values
+
+Programs act as a template for newly-created Values. So, for the different types of Values you create within Lightrail, it is recommended to create them from different Programs. Programs provide a template for all the Values that are created within that Program. Programs also enable organization and aggregated statistics on Values. Programs makes it really easy to see how a certain promotion you’re running is performing. Programs can be created through the API or web app.
```
Comment (jamietotten) [inline docs/object-model.md]: "Programs act as a template for newly-created Values." -> "Programs act as a template for created new Values."??

File: docs/currency-and-points.md (hunk 2 above)
Comment (Tim-Jordison) [reply docs/currency-and-points.md]: That seems reasonable, I'll update.

File: docs/object-model.md (hunk 3 above)
Comment (Tim-Jordison) [reply docs/object-model.md]: I'll update.

File: docs/welcome.md (hunk 6 above)
Comment (Tim-Jordison) [reply docs/welcome.md]: Hmm. I like that it's more specific but it feels a little confrontational. 

What about:
> Lightrail is an API-first tool that enables you to customize your payment experience. 

Build-in seems pretty implied by saying "API-first".

File: docs/wallet.md (hunk 5 above)
Comment (Tim-Jordison) [reply docs/wallet.md]: Should we do this and just assume it works?

Comment (jamietotten) [reply docs/wallet.md]: Yes

**Comment by jamietotten** [comment]
Merging into master in order to deploy the web app / docs changes to correspond.

**Pull request merged**